### PR TITLE
Tweak Mink tests, part 2

### PR DESCRIPTION
**Pull request by EreMaijala**
Improves findCssAndSetValue and uses it more. This should alleviate some full test retries. Also fixes SSO test to not leave a user dangling in the database.

### CODE REVIEW

File: module/VuFind/tests/integration-tests/src/VuFindTest/Mink/LibraryCardsTest.php [hunk 1]
```diff
@@ -224,7 +224,7 @@ public function testSwitchingCards(): void
 
         // Switch to the second card; we can't currently use findCssAndSetValue() here because
         // it conflicts with the behavior of jumpMenu.
-        $this->findCss($page, '#library_card')->setValue($card2Value);
+        $this->findCssAndSetValue($page, '#library_card', $card2Value);
```
Comment (demiankatz) [inline module/VuFind/tests/integration-tests/src/VuFindTest/Mink/LibraryCardsTest.php]: I think we need to revert this change -- see the comment above, indicating that findCssAndSetValue doesn't work here because of the jumpMenu behavior. Maybe we can move this change over to the remove jumpmenu PR but revert it here, since it's causing problems.

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @EreMaijala -- just one small problem; one test kept triggering retries in my environment. See below for the cause.

File: module/VuFind/tests/integration-tests/src/VuFindTest/Mink/LibraryCardsTest.php (hunk 1 above)
Comment (EreMaijala) [reply module/VuFind/tests/integration-tests/src/VuFindTest/Mink/LibraryCardsTest.php]: Oops, I didn't notice that one. I was able to reproduce the exception, so I added an option to skip the verification but continue to use findCssAndSetValue with its exception handling for added security.

**Review by demiankatz — APPROVED** [review]
Thanks, @EreMaijala -- the improvements look good, and tests are passing on my end!
